File: app/src/main/java/com/machiav3lli/derdiedas/ui/SettingsActivity.java
package com.machiav3lli.derdiedas.ui;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.machiav3lli.derdiedas.R;
import com.machiav3lli.derdiedas.databinding.ActivitySettingsBinding;

public class SettingsActivity extends AppCompatActivity {

    private ActivitySettingsBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivitySettingsBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        getSupportFragmentManager().beginTransaction().add(R.id.fragment_container, new SettingsFragment()).commit();
        binding.back.setOnClickListener(v -> onBackPressed());
    }
}

File: app/src/main/java/com/machiav3lli/derdiedas/ui/SettingsActivity.kt
package com.machiav3lli.derdiedas.ui

import android.os.Bundle
import androidx.activity.compose.setContent
import com.machiav3lli.derdiedas.ui.theme.AppTheme

class SettingsActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContent {
            AppTheme {
                SettingsScreen(
                    onBackPressed = { finish() }
                )
            }
        }
    }
}